### PR TITLE
add two files for hash and vid

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+In **Nebula Graph**, vertices are identified with vertex identifiers (i.e. `VID`s). 
+The VID can be an int64 or a fixed length string.
+When inserting a vertex, you must specify a `VID` for it. 
+
+You can also call `hash()` to generate an int64 VID if the graph has less than one billion vertices.
+
+> `VID` must be unique in a graph space.
+
+That is, in the same graph space, two vertices which have the same `VID` are considered as the same vertex.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md:15]: which have ==> that have

File: docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+In addition, one `VID` can have multiple `TAG`s. E.g., One person (`VID`) can have two roles (`tags`).
+
+Two `VID`s in two different graph spaces are totally independent of each other.
+
+## Partition ID
+
+When inserting into **Nebula Graph**, vertices and edges are distributed across different partitions. And the partitions are located on different machines. If you want some certain vertices to locate on the same partition (i.e., on the same machine), you can control the generation of the `VID`s by using the following [formula / code](https://github.com/vesoft-inc/nebula-common/blob/master/src/common/clients/meta/MetaClient.cpp).
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md:23]: If you want certain vertices
Remove "some"

File: docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+In the preceding formula,
+
+- `%` is the modulo operation.
+- `numParts` is the number of partition for the graph space where the `VID` is located, namely the value of `partition_num` in the [CREATE SPACE](../9.space-statements/1.create-space.md) statement.
+- `pId` is the ID for the partition where the `VID` is located.
+
+For example, if there are 100 partitions, the vertices with `VID` 1, 101, 1001 will be stored on the same partition.
+
+But, the mapping between the `partition ID` and the machine address are random. Therefore, you can't assume that any two partitions are located on the same machine.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/20.appendix.md/vid-partition.md:52]: The mapping **is** random.

**Review by Amber1990Zhang — COMMENTED** [review]
LGTM. pls update the yml file.

**Review by randomJoe211 — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**